care/facility/models/patient.py

```diff
@@ -149,6 +156,12 @@ class TestTypeEnum(enum.Enum):
         choices=RationCardCategory, null=True, max_length=8
     )
 
+    mobility_status = models.IntegerField(
+        choices=MobilityStatus,
+        default=MobilityStatus.INDEPENDENTLY_MOBILE,
```

Comment:
We will need an unknown option as well, we cannot just assume that everyone in care is under this option.